charts/openshift-metering/values.yaml

```diff
@@ -544,6 +544,16 @@ presto:
             useInstanceCredentials: null
 
         extraConnectorFiles: []
+        prometheus:
+          enabled: true
```

Comment:
If we're enabling the Prometheus-connector by default for OCP, and set `enabled: true` in the OCP enabled overrides dictionary, then we should disable this connector by default, in the MeteringConfig values.

Reply:
Done in 8962ea0b